### PR TITLE
Document that begin ... end use

### LINKED ISSUE #1863: Address `begin .. end` expressions
- [ ] Values & functions, `begin end` is just `( ... )`. Add explanation of this around line 290 in https://github.com/ocaml/ocaml.org/blob/main/data/tutorials/language/0it_00_values_functions.md in the applying functions section. Make a reference to another explanation in Imperative & Mutability.
- [ ] Imperative & Mutability (https://github.com/ocaml/ocaml.org/blob/main/data/tutorials/language/0it_06_imperative.md), what's where it's meant to be used

### PR DESCRIPTION
**Pull request by rmeis06**
Hello! This should resolve https://github.com/ocaml/ocaml.org/issues/1863

@cuihtlauac  @SaySayo  Does this look correct to yinz?

### CODE REVIEW

**Review by cuihtlauac — APPROVED** [review]

**Review by christinerose — APPROVED** [review]

File: data/tutorials/language/0it_00_values_functions.md [hunk 1]
```diff
@@ -294,6 +294,13 @@ The `max` function returns the largest of its two arguments, which are:
 - `42`, the result of `21 * 2`
 - `713`, the result of `int_of_string "713"`
 
+When creating subexpressions, it is also possible to use a `begin ... end` syntax. These are equivalant to using a `( ... )` syntax. As such, the above could also be rewritten and get the same result:
```
Comment (cuihtlauac) [inline data/tutorials/language/0it_00_values_functions.md]:
```suggestion
When creating subexpressions, using a `begin ... end` syntax is also possible. This is the same as using brackets `( ... )`. As such, the above could also be rewritten and get the same result:
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: data/tutorials/language/0it_00_values_functions.md [hunk 2]
```diff
@@ -294,6 +294,13 @@ The `max` function returns the largest of its two arguments, which are:
 - `42`, the result of `21 * 2`
 - `713`, the result of `int_of_string "713"`
 
+When creating subexpressions, using a `begin ... end` syntax is also possible. This is the same as using brackets `( ... )`. As such, the above could also be rewritten and get the same result:
```
Comment (cuihtlauac) [inline data/tutorials/language/0it_00_values_functions.md]:
```suggestion
When creating subexpressions, using `begin ... end` is also possible. This is the same as using brackets `( ... )`. As such, the above could also be rewritten and get the same result:
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Review by cuihtlauac — APPROVED** [review]
Thanks, @rmeis06, this is nice.

@christinerose : is line 297 ok for you?

**Comment by christinerose** [comment]
> Thanks, @rmeis06, this is nice.
> 
> @christinerose : is line 297 ok for you?

LGTM!

**Pull request merged**